docs/syntax-reference.md

```diff
@@ -528,10 +528,10 @@ $ cat ksql.properties
 auto.offset.reset=earliest
 
 # Start KSQL in standalone mode with the custom properties above
-$ ksql-cli local --properties-file ./ksql.properties
+$ ksql-cli local ./ksql.properties
```

Comment:
You will need the --properties-file flag. It would not work without it.

Reply:
@hjafarpour didn't you say differently here: https://confluentinc.atlassian.net/browse/KSQL-351?focusedCommentId=42416&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-42416

Reply:
@joel-hamill that was for the ksql-server. The above command is for ksql-cli. The server does not need `--properties-file` but cli needs it.